package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/screenshot-glb",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
@sleroux I saw you've done `patch` updates for these. But I wonder if they should be Minor instead?

Reply:
Minor makes more sense considering it's a larger change than a hot fix. No reasoning for using patch other than using the incorrect mental model for htis.